test/__tests__/data/kustomer_input.json

```diff
@@ -785,5 +792,69 @@
       },
       "originalTimestamp": "2023-01-10T15:32:56.409Z"
     }
+  },
+  {
+    "destination": {
+      "Config": {
+        "apiKey": "WJqij952gdy251",
```

Comment:
Is this real `apiKey` ?

Reply:
No, but I will put some dummy value whatsoever.